src/main/resources/templates/macros/json/steps.vm

```diff
@@ -9,6 +9,9 @@
   <div id="steps-$stepsId" class="inner-level collapse collapsable-details #if (!$element.getStatus().isPassed() || $expand_all_steps) in #end">
     #foreach($step in $element.getSteps())
       <div class="step">
+        #foreach($comment in $step.getComments())
+          <div class="description indention">$comment</div>
```

Comment:
why 'description' class for comment? this is misleading
new class should be defined

Reply:
please add screenshot here to see how comments are rendered

Reply:
@damianszczepanik your remarks have been taken into account. For the screenshots, I was not sure if you meant here in the PR. But instead I updated the screenshot integrated in the README.

Reply:
Based on https://raw.githubusercontent.com/jfougere/cucumber-reporting/feature/support-step-comments/.README/feature-failed.png I don't know if the comment is related to hook or step because of indention.

Reply:
For me it's clear thanks to the indentation that the comments belongs to the step and not the hook else it would be moved in step to the right.

Were you thinking of another display ?